### PR TITLE
#84: Small code updates and typing fixes to resolve linter complaints and fix tests

### PR DESCRIPTION
**Pull request by mikolevy**

### CODE REVIEW

File: src/screens/runPlan/runPlanList/PlanItemListItem.tsx [hunk 1]
```diff
@@ -1,9 +1,9 @@
 import React from 'react';
-import { StyleSheet, TouchableHighlight } from 'react-native';
+import {StyleSheet, TouchableHighlight, ViewStyle} from 'react-native';
 
 import { Card } from 'components';
 import { PlanItem } from 'models';
-import { palette } from 'styles';
+import {palette, typography} from 'styles';
```
Comment (magasol) [inline src/screens/runPlan/runPlanList/PlanItemListItem.tsx]: Is it a differenece when spaces are used or omitted at the ends of { }?

File: src/screens/runPlan/runPlanList/PlanItemListItem.tsx [hunk 2]
```diff
@@ -55,22 +56,18 @@ const styles = StyleSheet.create({
   },
   nameTextColor: {
     color: palette.textBlack,
+      flex: 1,
```
Comment (magasol) [inline src/screens/runPlan/runPlanList/PlanItemListItem.tsx]: should indent be 2 or 4? it's different for different styles

Comment (mikolevy) [reply src/screens/runPlan/runPlanList/PlanItemListItem.tsx]: 2 in general :) Will fix this

File: src/screens/runPlan/runPlanList/PlanItemListItem.tsx (hunk 1 above)
Comment (mikolevy) [reply src/screens/runPlan/runPlanList/PlanItemListItem.tsx]: Not a difference - PyCharm put imports in this way by default :)

**Pull request merged**